src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -91,17 +93,20 @@ public ExportPanel(TerminationFuture terminationFuture, ExportForms forms, Expor
 
     form.onExport(() -> new Thread(() -> {
       List<String> errors = export();
+      int exportedFormsAmount = forms.getSelectedForms().size();
       if (!errors.isEmpty()) {
+        analytics.event("Export", "Export", "Configuration errors", exportedFormsAmount);
```

Comment:
A small point. Feel free to ignore. I would code the isEmpty case first, for two reasons. I’ll bet you know what they are.

Reply:
I don't reject the idea of not negating that condition